bundles/org.openhab.binding.ism8/README.md

```diff
@@ -0,0 +1,161 @@
+The system works in a way that the ISM8 connects to a partner and sends from time to time an update. The frequency depends on the change of the values. This binding is listening to those messages.
+After the first connection there is an active command send to the ISM8 in order to receive all available data points.
+The manual of the ISM8 can be downloaded from the supplier (https://www.wolf.eu/fileadmin/Wolf_Profi/Downloads/Montage-Bedienungsanleitungen/Regelungen/Zubehoer/3064356_201611_ISM8i_Montage-u.Bedienungsanleitung.pdf) 
+
+## Supported Things
+
+_This binding does only support one Thing - the Ism8-Device._
+
+## Discovery
+
+_Auto-discovery is not supported._
+
+
+## Thing Configuration
+
+The intension was to have a generic ISM8 binding in order to offer the full flexibilty for the different heating systems.
+For this reason you need to create a Thing configuration, where basically only the port is required next to the channel configuration.
+(Thing ism8:device:heater "Wolf Heizung" [portNumber=12004])
+
+## Channels
+
+You can use any channel supported by the ISM8 as data point. Please have a look at the official manual from Wolf.
+Within this document you'll find a table containing all supported data points. The available data points are depending on your heating system configuration. 
+The Ism8 does currently support 4 different devices at the same moment of time (e.g. CGB-2, CWL Excellent, Solar, ...).
+
+Once you have an overview of your heating system set you can start to create the channels accordingly.
+Each channel should be created in the following way:
+
+	| Type   | Name    | Description                | Configuration   |
+	|--------|---------|----------------------------|-----------------|
+	| Number | DpId004 | "Kesseltemperatur"         | id, type, write |
+
+Type:
+
++ Switch use for boolean values
++ Number use for any number
++ Other types may work as well.
+
+Name:
+
++ Put here any name you'd like. This name is used for creating the binding.
+
+
+Description:
+
++ Put here any description you'd like. This helps you to identify the channels in the PaperUI
+
+
+Configuration:
+
++ id=1            - Please enter here the ID of the data point you'd like to map to this channel
++ type="1.001"    - Please enter here the knx type of the data point. You can find the data type in the Wolf ISM8 document.
++ write=true      - This parameter defines if the channel is bidirectional. The parameter is optional and by default false. 
+
+Note:
+Not all available types of the ISM8 interface are fully supported, but this can be extended.
+For the moment the following data types are implemented:
+
++ DPT-Bool:         "1.001", "1.002", "1.003", "1.009"
```

Comment:
Does the user know what DPT stands for?

Reply:
I think so, as I was using the same naming as in the ISM8 manual. At the end does a user need to look into the manual to identify the specific heating system and make a selection of values to be viewed or controlled via OpenHab.